omnibus/files/private-chef-cookbooks/private-chef/recipes/opscode-erchef.rb

```diff
@@ -59,6 +59,12 @@
   notifies :restart, 'runit_service[opscode-erchef]' unless backend_secondary?
 end
 
+template "/opt/opscode/embedded/service/opscode-erchef/vm.args" do
```

Comment:
I saw that. I didn't know the reasoning behind symlinking instead of just writing the file. If you think thats the right thing to do, then I'll do that!